migrate2rocky/migrate2rocky.sh

```diff
@@ -351,10 +351,13 @@ collect_system_info () {
     # Check the efi mount first, so we can bail before wasting time on all these
     # other checks if it's not there.
     if [[ $update_efi ]]; then
-	declare -g efi_mount
+	local efi_mount
+	declare -g efi_disk efi_partition
 	efi_mount=$(findmnt --mountpoint /boot/efi --output SOURCE \
 	    --noheadings) ||
 	    exit_message "Can't find EFI mount.  No EFI  boot detected."
+	efi_disk=/dev/$(lsblk -no pkname "$efi_mount")
+	efi_partition=$(<"/sys/block/$efi_disk/${efi_mount##*/}/partition")
```

Comment:
```
[root@server ~]# efi_partition=$(<"/sys/block/$efi_disk/${efi_mount##*/}/partition")
-bash: /sys/block//dev/sdb/sdb2/partition: No such file or directory
```

An additional `/dev` needs to be stripped,

```suggestion
	efi_partition=$(<"/sys/block/${efi_disk##*/}/${efi_mount##*/}/partition")
```

Reply:
Thanks, good spotting.